elytron-oidc-client/subsystem/src/main/resources/org/wildfly/extension/elytron/oidc/LocalDescriptions.properties

```diff
@@ -1,144 +1,142 @@
-elytron-oidc-client.secure-deployment.public-key-cache-ttl=Maximum time the downloaded public keys are considered valid. When this time reach, the adapter is forced to download public keys from elytron-oidc-client server
-elytron-oidc-client.secure-deployment.ignore-oauth-query-parameter=disable query parameter parsing for access_token
+elytron-oidc-client.secure-deployment.add=Add a deployment to be secured by an OpenID Connect provider.
+elytron-oidc-client.secure-deployment.remove=Remove a deployment to be secured by an OpenID Connect provider.
+elytron-oidc-client.secure-deployment.realm=The Keycloak realm used for the authentication.
+elytron-oidc-client.secure-deployment.provider=The OpenID Connect provider used for the authentication.
+elytron-oidc-client.secure-deployment.realm-public-key=The public key of the OpenID provider realm in the PEM format. This is optional. It is not recommended to set it.
+elytron-oidc-client.secure-deployment.auth-server-url=The base URL of the Keycloak authorization server. This is Keycloak specific.
+elytron-oidc-client.secure-deployment.provider-url=The URL of the OpenID provider.
+elytron-oidc-client.secure-deployment.disable-trust-manager=Whether or not to make use of a trust manager when communicating with the OpenID provider over HTTPS. This is optional. This should be set to 'true' only during development and never in production as it disables verification of SSL certificates.
+elytron-oidc-client.secure-deployment.ssl-required=Whether the communication with the OpenID provider should be over HTTPS. Valid values are: "all" - to always require HTTPS, "external" - to only require HTTPS for external requests, "none" - if HTTPS is not required. This should be set to "all" in production environments.
+elytron-oidc-client.secure-deployment.confidential-port=The confidential port used by the OpenID provider when communicating securely over SSL/TLS.
+elytron-oidc-client.secure-deployment.allow-any-hostname=If 'true', hostname verification will be skipped when communicating with the OpenID provider over HTTPS. This can be useful in testing environments. This should never be set to 'true' in production environments as it disables verification of SSL certificates.
+elytron-oidc-client.secure-deployment.truststore=The path to the truststore to be used when communicating with the OpenID provider over HTTPS. If you prefix the path with "classpath:", the trust store will be obtained from the deployment's classpath.
+elytron-oidc-client.secure-deployment.truststore-password=The password of the truststore.
+elytron-oidc-client.secure-deployment.connection-pool-size=The connection pool size to be used when communicating with the OpenID provider.
+elytron-oidc-client.secure-deployment.resource=Specify the name of the application secured with OIDC. Alternative to the client-id.
```

Comment:
This is the client identifier for the application. This is the identifier that has been registered with the authorization server.